### ISSUE TITLE
CRUD operation of Category

### ISSUE THREAD

**Issue by dhwaj1902** [Learn-Write-Repeat/LWR-WebApp#19]
Create CRUD operation for modal ``blog_category`` which has:
- name
- position

- [x] DCP21

**Comment by sanyogchangmai** [comment]
@dhwaj1902 can you explain the issue. Do we need to add a extra property blog_category to the existing blog model?

**Comment by dhwaj1902** [comment]
> @dhwaj1902 can you explain the issue. Do we need to add a extra property blog_category to the existing blog model?

No, you need to add create another modal called ``blog_category``

**Comment by dhwaj1902** [comment]
@sanyogchangmai are you working on this?

**Comment by sanyogchangmai** [comment]
Ok I will work on it and also please tell what properties will this model have?
Also why we need to create a separate model for this cant we just add an extra property in blogs model so that user can write or select the category while they submit the blog?

**Comment by dhwaj1902** [comment]
We will do that, what we do is we create a category model and give category_id to the blog too which the blog is suitable.

**Comment by sanyogchangmai** [comment]
Ok got it .

**Comment by sanyogchangmai** [comment]
Can you tell me what properties will the blog_category model have?

**Comment by sanyogchangmai** [comment]
@dhwaj1902 I am stuck with n error since yesterday its something related to slug. Here it is.
![problem slug](https://user-images.githubusercontent.com/76730995/127737724-bdc61d83-0274-45fd-89d9-f24df5895268.JPG)


I wrote the code exactly as it was for the blog models including the routes and controllers. I made the changes that were required. But when I test the API I get the above error when I tried removing the slug it would won't save the data but it gets saved only with the autogenerated _id and timestamps in MongoDB.
I am stuck with the POST part rest every other route works fine.
Plz have a look and see if you get any idea.
